src/IssueViz.Security/HotspotsList/TableDataSource/HotspotsTableDataSource.cs

```diff
@@ -40,17 +40,19 @@ internal interface IHotspotsStore
     [PartCreationPolicy(CreationPolicy.Shared)]
     internal sealed class HotspotsTableDataSource : IHotspotsStore, IIssueLocationStore, ITableDataSource, IDisposable
     {
+        private readonly IHotspotTableEntryFactory tableEntryFactory;
         private readonly ITableManager tableManager;
         private readonly ISet<ITableDataSink> sinks = new HashSet<ITableDataSink>();
-        private readonly List<HotspotTableEntry> tableEntries = new List<HotspotTableEntry>();
+        private readonly List<ITableEntry> tableEntries = new List<ITableEntry>();
```

Comment:
So it would be easier to test, there's no actual need to know that it's a `HotspotTableEntry`